### ISSUE TITLE
Support for Lunar/Badlion client capes

### ISSUE THREAD

**Issue by RaptaG** [CaelTheColher/Capes#54]
Fabric Capes already supports LabyMod capes and supporting also Lunar and Badlion client's capes would be awesome!

**Comment by NolnobLIVE** [comment]
> The cape server URLs or IP addresses are needed to do that.

so, Lunar/Badlion capes should work, or did i missunderstood something?

**Comment by shimizu-izumi** [comment]
Yes, they should work. I will try to get them working, and if I get them to work, I will open a PR

**Comment by shimizu-izumi** [comment]
Badlion is completely obfuscated and for Lunar the class and method names are unreadable, so the use of a tool like Wireshark would be needed to get the domain/IP address

**Comment by Cuteistfox** [comment]
i belive cosmetica did it so you can read ther code for lunar

**Comment by kaxlabs** [comment]
Feather client capes would also be a nice addition.
